compile failure in local build

I just created a new branch based on the latest integration and when I ran gradlew assemble it failed with this:

> Task :com.ibm.ws.container.service:compileJava FAILED
/Users/nottinga/Documents/github/open-liberty/dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/metadata/internal/MetaDataServiceImpl.java:46: error: cannot find symbol
    private final MethodMetaDataManager methodMetaDataManager = new MethodMetaDataManager("methodMetaDataListeners");
                  ^
  symbol:   class MethodMetaDataManager
  location: class MetaDataServiceImpl
/Users/nottinga/Documents/github/open-liberty/dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/metadata/internal/MetaDataServiceImpl.java:46: error: cannot find symbol
    private final MethodMetaDataManager methodMetaDataManager = new MethodMetaDataManager("methodMetaDataListeners");
                                                                    ^
  symbol:   class MethodMetaDataManager
  location: class MetaDataServiceImpl
2 errors

FAILURE: Build failed with an exception.

I spoke to @aguibert and he says he has the same issue sometimes and assumed it was because he keeps switching Java versions, but I'm not in that situation.

--- a/dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/metadata/internal/MethodMetaDataManager.java
+++ b/dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/metadata/internal/MethodMetaDataManager.java
@@ -15,7 +15,7 @@ import com.ibm.ws.container.service.metadata.MetaDataException;
 import com.ibm.ws.container.service.metadata.MethodMetaDataListener;
 import com.ibm.ws.runtime.metadata.MethodMetaData;
 
-class MethodMetaDataManager extends MetaDataManager<MethodMetaData, MethodMetaDataListener> {
+public class MethodMetaDataManager extends MetaDataManager<MethodMetaData, MethodMetaDataListener> {
     MethodMetaDataManager(String listenerRefName) {
         super(listenerRefName);
     }

